### PR TITLE
fix: make the custom snackbar as default

### PR DESCRIPTION
**Pull request by UncleBill**
| On light mode | On dark mode |
| -- | -- |
| image | image |

## Change anchorOrigin (from top center)

image

### CODE REVIEW

File: packages/theme/src/Components/Snackbar/index.tsx [hunk 1]
```diff
@@ -224,3 +239,25 @@ export const CustomSnackbarProvider = memo<SnackbarProviderProps>((props) => {
         />
     )
 })
+
+export function useShowCostomSnackbar() {
```
Comment (nuanyang233) [inline packages/theme/src/Components/Snackbar/index.tsx:243]:
```suggestion
export function useShowCustomSnackbar() {
```

**Review by nuanyang233 — CHANGES_REQUESTED** [review]

File: packages/theme/src/Components/Snackbar/index.tsx [hunk 2]
```diff
@@ -224,3 +239,25 @@ export const CustomSnackbarProvider = memo<SnackbarProviderProps>((props) => {
...
+    const snackbar = useSnackbar()
+    const showSnackbar = useCallback(
+        (
+            message: string,
+            options: Partial<CustomSnackbarContentProps & { persist: boolean }> = {
+                variant: 'default',
```
Comment (nuanyang233) [inline packages/theme/src/Components/Snackbar/index.tsx:249]: Perhaps in most scenarios, `success` is better as the default value

File: packages/theme/src/Components/Snackbar/index.tsx [hunk 3]
```diff
@@ -224,3 +239,25 @@ export const CustomSnackbarProvider = memo<SnackbarProviderProps>((props) => {
         />
     )
 })
+
+export function useShowCostomSnackbar() {
+    const snackbar = useSnackbar()
+    const showSnackbar = useCallback(
```
Comment (nuanyang233) [inline packages/theme/src/Components/Snackbar/index.tsx:245]:
```suggestion
   return useCallback(
```

File: packages/theme/src/Components/Snackbar/index.tsx (hunk 2 above)
Comment (UncleBill) [reply packages/theme/src/Components/Snackbar/index.tsx:257]: It's all based on the design.

File: packages/theme/src/Components/Snackbar/index.tsx (hunk 1 above)
Comment (UncleBill) [reply packages/theme/src/Components/Snackbar/index.tsx]: Fixed

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**